.gitignore

```diff
@@ -102,9 +102,10 @@ install/yaml/
 tmp/
 
 # Terraform context
-.terraform
+.terraform*
 *.tfstate.backup
 
+
```

Comment:
nit: extra whitespace here?

Reply:
thanks, I don't know why I put it there...